scintilla/src/Editor.cxx

```diff
@@ -4849,17 +4851,22 @@ void Editor::ButtonMoveWithModifiers(Point pt, unsigned int, int modifiers) {
 			hotSpotClickPos = INVALID_POSITION;
 		}
 
+		if ((vs.fixedColumnWidth > 0) && PointInSelMargin(pt)) {
+			SetHoverIndicatorPosition(INVALID_POSITION);
+		}
 	} else {
 		if (vs.fixedColumnWidth > 0) {	// There is a margin
 			if (PointInSelMargin(pt)) {
 				DisplayCursor(GetMarginCursor(pt));
 				SetHotSpotRange(nullptr);
+				SetHoverIndicatorPosition(INVALID_POSITION);
 				return; 	// No need to test for selection
 			}
 		}
 		// Display regular (drag) cursor over selection
 		if (PointInSelection(pt) && !SelectionEmpty()) {
 			DisplayCursor(Window::cursorArrow);
+			SetHoverIndicatorPosition(INVALID_POSITION);

```

Comment:
One more stuff, in order to make future Scintilla updating easier, could you add the following comment:
// Fix the following issue:
// https://github.com/notepad-plus-plus/notepad-plus-plus/issues/8647
// It could be fixed in Scintilla release superior to v4.4.4 - to verify.

Reply:
Please add whole link instead of issue number.